src/components/Tooltip/Tooltip.test.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react'
 import { mount } from 'enzyme'
+import { render, waitFor, screen, fireEvent } from '@testing-library/react'
```

Comment:
One more thing, for consistency with other tests use `import user from '@testing-library/user-event'` instead of `fireEvent`, then `user.click(YOUR_ELEMENT)`